### PR TITLE
Adding proposal clarifications

### PR DESCRIPTION
**Pull request by VoteWizard**

### CODE REVIEW

File: governance/votes/Executive vote - September 13, 2024.md
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:51]:
```suggestion
### New Contract ESM Exclusion
```

File: governance/votes/Executive vote - September 13, 2024.md
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:56]:
```suggestion
The [Emergency Shutdown Module (ESM)](https://docs.makerdao.com/smart-contract-modules/shutdown/emergency-shutdown-module) will **not** be authorized in any new contracts deployed for this proposal.
```

**Review by MakerJanSky — COMMENTED** [review]
Looking good, VoteWizard. I will approve this PR once Ecosystem Approval for the New Contract ESM Exclusion has been linked.

File: governance/votes/Executive vote - September 13, 2024.md
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:191]:
```suggestion
- **Authorization**: [Ecosystem Approval 1](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Ecosystem Approval 2](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/4), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi) ,[Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
```

File: governance/votes/Executive vote - September 13, 2024.md
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:53]:
```suggestion
- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/4)
```

File: governance/votes/Executive vote - September 13, 2024.md
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:191]:
```suggestion
- **Authorization**: [Ecosystem Approval 1](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Ecosystem Approval 2](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/4), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
```

**Review by MakerJanSky — APPROVED** [review]
Nice work, all.

**Pull request merged**
